extension/oidcextension/oidc_server_test.go

```diff
@@ -33,7 +33,7 @@ import (
 	"time"
 )
 
-// oidcServer is an overly simplified OIDC mock server, good enough to sign the tokens required by the test
+// oidcServer is an overly simplified Config mock server, good enough to sign the tokens required by the test
```

Comment:
probably revert this search-and-replace?

Reply:
interesting because I let goland do that with a refactor